docs/source/configuration.mdx

```diff
@@ -0,0 +1,133 @@
+# Configuration
```

Comment:
Maybe name it `environment_variables` as this page is focused solely on that?

Reply:
Otherwise I would expect this page to also contain info about stuff such as `lfs-enable-largefiles` which is also part of the "Configuration" but out of scope for this specific reference

Reply:
Great job documenting all these environment variables! 

I'd consider moving this page to the Reference section, mention a few of the most important (or common) ones on the Installation page, and linking to the rest of them. While this is useful info, I don't think it is necessarily important for the majority of users who are getting started.

Reply:
I moved the configuration page to `package_references/environment_variables` in https://github.com/huggingface/huggingface_hub/pull/1233/commits/7c145cf83e1689b867d7010c50482265dd055f00. Thanks for the feedback !